### PR TITLE
Small fixes for groundtruth generation pipeline

### PR DESCRIPTION
**Pull request by rsoussan**
Fixed some issues that caused faults in the groundtruth pipeline when running multiple bags concurrently.  Also fixed a path issue where a dependent script was pointing to astrobee/astrobee to the config path instead of the updated astrobee/src/astrobee. Finally, added merge_maps from sparse mapping to the cmake file (this might have been an accidental relic of the catkinizing pr).

### CODE REVIEW

File: tools/graph_bag/scripts/utilities.py [hunk 1]
```diff
@@ -77,6 +77,7 @@ def load_dataframe(files):
 
 
 def run_command_and_save_output(command, output_filename):
+    print(command)
```
Comment (bcoltin) [inline tools/graph_bag/scripts/utilities.py:80]: did you mean to leave this?

**Review by bcoltin — APPROVED** [review]

**Comment by bcoltin** [comment]
Looks good once you fix lint (and double check the print is what you wanted)

File: tools/graph_bag/scripts/utilities.py (hunk 1 above)
Comment (rsoussan) [reply tools/graph_bag/scripts/utilities.py:80]: Oh yeah I did, but I'll make it optional and pass a bool for printing.  I found it helpful to have the command being run printed to the file but I can see how other times people wouldn't want that

Comment (trey0) [reply tools/graph_bag/scripts/utilities.py:80]: The logging module may be a good way to handle that.

**Review by marinagmoreira — APPROVED** [review]

**Review by marinagmoreira — APPROVED** [review]

**Pull request merged**